src/khoj/interface/web/agent.html

```diff
@@ -0,0 +1,286 @@
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0 maximum-scale=1.0">
+        <title>Khoj - Agents</title>
+
+        <link rel="icon" type="image/png" sizes="128x128" href="/static/assets/icons/favicon-128x128.png?v={{ khoj_version }}">
+        <link rel="manifest" href="/static/khoj.webmanifest?v={{ khoj_version }}">
+        <link rel="stylesheet" href="/static/assets/khoj.css?v={{ khoj_version }}">
+    </head>
+    <script type="text/javascript" src="/static/assets/utils.js?v={{ khoj_version }}"></script>
+    <body>
+        <!--Add Header Logo and Nav Pane-->
+        {% import 'utils.html' as utils %}
+        {{ utils.heading_pane(user_photo, username, is_active, has_documents) }}
+        <div id="agent-metadata-wrapper">
+            <div id="agent-metadata">
+                <div id="agent-avatar-wrapper">
+                    <div id="agent-settings-header">Agent Settings</div>
+                </div>
+                <div class="divider"></div>
+                <div id="agent-data-wrapper">
+                    <div id="agent-avatar-wrapper">
+                        <img id="agent-avatar" src="{{ agent.avatar }}" alt="Agent Avatar">
+                        <input type="text" id="agent-name-input" value="{{ agent.name }}" {% if agent.creator_not_self %} disabled {% endif %}>
+                    </div>
+                    <div id="agent-instructions">Instructions</div>
+                    <div id="agent-tuning">
+                        <p>{{ agent.tuning }}</p>
```

Comment:
Should this be updated to use `agent.personality` now?

```suggestion
                    <div id="agent-tuning">
                        <p>{{ agent.personality }}</p>
```